resources/js/components/FormField.vue

```diff
@@ -162,6 +164,25 @@ export default {
         fieldLabel() {
             return this.field.singularLabel || this.field.name
         },
+
+        isAttributeReadOnly()
+        {
```

Comment:
You can put the `{` on the same row with the `()` as in the method fieldLabel. This applies to methods below as well